prog/test/hetzner_server.rb

```diff
@@ -145,4 +159,8 @@ def hetzner_api
   def vm_host
     @vm_host ||= VmHost[frame["vm_host_id"]]
   end
+
+  def default_boot_images
+    @default_boot_images ||= YAML.load_file("config/e2e_boot_images.yml")
```

Comment:
We can determine which test cases to run, so we may not need all the images. Rather than using a static file, we can identify the necessary images based on the `options[:test_cases]` value and download only those.

Reply:
Makes sense, added this